src/components/app/rest-ly/History/HistoryItem.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HistoryItem = ({ method, url, id }) => {
+  return (
+    <article id={id}>
+      <p id={id}>{method}</p>
+      <span id={id}>{url}</span>
```

Comment:
You don't want different tags to have the same id - you should use a class for that